lib/remote.rb

```diff
@@ -0,0 +1,114 @@
+require 'ostruct'
+require 'optparse'
+
+# rubocop:disable Rails/Output
+# rubocop:disable Rails/Exit
+class Remote
+  def run(argv)
+    config = parse(argv)
+
+    execute(config)
+  end
+
+  def parse(argv)
+    config = default_config
+
+    parser = build_parser(config)
+    config.command = extract_trailing_options(argv)
+    parser.parse!(argv)
+    parse_positional_aguments(config, argv)
+
+    if config.show_help || !config.stage
+      puts parser
```

Comment:
Ah, good to know. Since this is never run inside of a rails environment, seemed fine to just rubocop ignore?